clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -749,8 +750,8 @@ static CompoundStmt *CreateOpenCLKernelBody(Sema &S,
 
         llvm::SmallVector<Expr *, 4> ParamStmts;
         const auto *Proto = cast<FunctionProtoType>(Method->getType());
-        S.GatherArgumentsForCall(SourceLocation(), Method, Proto, 0,
-                                 ParamDREs, ParamStmts);
+        S.GatherArgumentsForCall(SourceLocation(), Method, Proto, 0, ParamDREs,
```

Comment:
What changed here?  I realize that the formatting tool probably got this, but please don't change this unless you do it in a patch specifically to fix the formatting.